wodles/aws/aws_s3.py

```diff
@@ -74,6 +74,13 @@ def main(argv):
                 bucket_type = buckets_s3.server_access.AWSServerAccess
             else:
                 raise Exception("Invalid type of bucket")
+            for region in options.regions:
+                try:
+                    if region not in aws_tools.ALL_REGIONS:
+                        raise ValueError(f"Invalid region '{region}'")
+                except ValueError as exc:
+                    aws_tools.debug(f"+++ ERROR: {exc}", 1)
+                    exit(22)
```

Comment:
The issue description mentions to include a warning when the region does not exists, I think it is because raising an error would cause the `test_regions` integration tests to never succeed, since the region used in some of them is `us-fake-1`. 

Also, I think the 6 integration tests that failed in the [checks](https://github.com/wazuh/wazuh/actions/runs/9420004162/job/25951065783?pr=23998) should pass after the changes from this issue.